source/parameters/bend.md

```diff
@@ -0,0 +1,193 @@
+%---------------------------------------------------------------------------------------------------
+(s:bend.params)=
+## BendP: Bend Parameters
+
+The `BendParams` stores the parameters that characterize the shape of a [`Bend`](#s:bend) element. 
+The only relavent shape parameter that is not in the `BendParams` is the
+length `L` which is in the `LengthParams`.
+
+The parameters of this group are:
+```{code} yaml
+bend_type         - Type of bend. Default: SECTOR.
+angle             - Reference bend angle.
+g                 - Reference bend strength = 1/radius.
+bend_field_ref    - Reference bend field.
+L_chord           - Chord length.
+tilt_ref          - Reference tilt.
+e1                - Entrance end pole face rotation.
+e2                - Exit end pole face rotation.
+e1_rect           - Entrance end pole face rotation.
+e2_rect           - Exit end pole face rotation.
+edge_int1         - Entrance end fringe field integral.
+edge_int2         - Exit end fringe field integral
+```
```

Comment:
There is redundancy in this choice of parameters, which provides a lot of flexibility to the user.  However, it also requires that consistency conditions be imposed, and it complicates the description of the standard.  For example, I don't see why parameters `(e1_rect, e2_rect)` are needed.  Why not use `e1` and `e2` only, and define these with respect to the appropriate pole face associated with the given `bend_type`?  Likewise, do we need `L_chord` as a separate input if the bend `Length`, `angle`, and `1/radius` are all specified?  The same comment applies to the pair `g` and `bend_field_ref`.

Reply:
As you note, a user may have a rectangular bend and may want to therefore specify `e1_rect`, `e2_rect`, and `L_chord`. That said, it would indeed simplify things if these quantities where made to be not User settable. This should be discussed.

Reply:
I think this comment and the discussion that followed (regarding parameters specifying a Bend) may not yet be fully resolved.  This stimulated discussion about RBENDs and an exchange with Laurent Deniau, but I'm not sure how this was left.

Reply:
The discussion about what is User settable and what is not is mostly being left to later PRs.

Reply:
Thanks.  Just to clarify -- this comment is about the parameters needed to specify a Bend (and the case of RBENDS in particular) -- separate from the discussion about what is User settable.

Reply:
Note that as a result of the conversation with Laurent, I did change the PR to use `SBend` and `RBend` elements instead of just having `Bend` elements.